packages/search-ui/src/actions/a11yNotify.js

```diff
@@ -0,0 +1,29 @@
+import { announceToScreenReader } from "../A11yNotifications";
+
+/**
+ * Announces a specific message in `a11yNotificationMessages`
+ * to Search UI's screen reader live region.
+ *
+ * @param {string} messageFunc - key of a message function in `a11yNotificationMessages`
+ * @param {object} [messageArgs] - arguments to pass to the message function, if any
+ */
+export default function a11yNotify(messageFunc, messageArgs) {
+  if (!this.a11yNotifications) return;
```

Comment:
I actually really tend to like the `isX`/`hasY`/`canZ` nomenclature for naming boolean variables ([quick overview here](https://www.samanthaming.com/tidbits/34-better-boolean-variable-names)). I picked that up from my last workplace (which I think maybe adopted that from airbnb or something), and it's been pretty nice to work with.

That being said, something like that might be better for wider refactor/overhaul and is probably out of scope :)

For this current PR/comment - hmm, I don't feel super strongly either way. I'm fine with whatever, so happy to defer to you or @JasonStoltz on this one (or put it to a vote).

Reply:
I agree that `a11yNotifications` is not totally clear, something else would be better. As to what, that's up to you.

Reply:
I like the "boolean" prefix, Constance! We should definitely use it.

> That being said, something like that might be better for wider refactor/overhaul and is probably out of scope :)

I might misunderstand you here, are you suggesting to change the name of a variable later? If not, sorry! If yes, I would be totally ok with that if it was an internal variable, but since this name goes to the public API I think it would be better to change the name before the release. 

I don't have a preference regarding the name either. Just throwing a couple of options to consider: `areA11yNotificationsEnabled`, `areA11yNotifications[Turned]On`; or your option!

Reply:
@yakhinvadim No misunderstanding, I was totally thinking of something else with that comment, my bad!

I think between all the options I like `hasA11yNotifications` - it's the shortest/most succinct between `enabled`, `turnedOn` etc. while still conveying a boolean nature.

The only other suggestion I might throw out there would be `a11yNotificationsFlag`, which feels different enough but tied to `a11yNotificationMessages`. But overall I think I still like `has` a bit better.

Reply:
Yep, I like the 'has' option, let's go with that!

Reply:
Addressed in b9df4f8